core/trino-main/src/main/java/io/trino/sql/planner/StatisticsAggregationPlanner.java

```diff
@@ -113,6 +114,8 @@ private ColumnStatisticsAggregation createColumnAggregation(ColumnStatisticType
                 return createAggregation(QualifiedName.of("max"), input.toSymbolReference(), inputType, inputType);
             case NUMBER_OF_DISTINCT_VALUES:
                 return createAggregation(QualifiedName.of("approx_distinct"), input.toSymbolReference(), inputType, BIGINT);
+            case NUMBER_OF_DISTINCT_VALUES_SUMMARY:
+                return createAggregation(QualifiedName.of("approx_set_generic"), input.toSymbolReference(), inputType, HYPER_LOG_LOG);
```

Comment:
update the commit message

> Support approx_set as a column statistics aggregation

or add a commit description explaining why it's not the actual
approx_set what is being used.

Reply:
I updated the commit message and added a comment explaining why `$approx_set` is used